docs_espressif/en/additionalfeatures/project-configuration.rst

```diff
@@ -0,0 +1,82 @@
+Project Configuration Editor
+====================================
+
+To allow the user to have multiple configurations for the same project, the user can define several settings to produce different build results. For example, take a look at the `Multiple configuration tutorial <multiple_config>`_ which uses the ESP-IDF CMake build system `multi_config <https://github.com/espressif/esp-idf/tree/master/examples/build_system/cmake/multi_config>`_ example.
```

Comment:
For consistency in addressing the reader, I would suggest addressing him directly and not use the `user` word here